examples/tensorflow_converter/example_tensorflow_parser.cpp

```diff
@@ -0,0 +1,307 @@
+#include <iostream>
+#include <fstream>
+#include <sstream>
+#include <string>
+#include <map>
+#include "graph.pb.h"
+using namespace std;
+using namespace tensorflow;
+
+template<class in_value>
+void all2str(string & result, const in_value& t)
+{
+    ostringstream oss;
+    oss<<t;
+    result=oss.str();
+}
+
+/* This method is used as reference for bit operation for char* parsering
```

Comment:
Yeah I see, already removed. The IOS codes might be pushed tommorow as a standalone PR~